rholang/README.md

```diff
@@ -25,57 +27,28 @@ The language is still in the early stages of development, but for those who are
     * BNFC - must be built from [git](https://github.com/BNFC/bnfc) b0252e5f666ed67a65b6e986748eccbfe802bc17 or later
     * Scala
-4. Run `sbt bnfc:generate` to generate the lexer/parser. Re-run whenever you modify the grammar
-5. Run `sbt compile` to compile classes
-6. Run `sbt assembly` to build a stand-alone `.jar` file
+4. Run `sbt rholang/bnfc:generate` to generate the lexer/parser. Re-run whenever you modify the grammar
+5. Run `sbt rholang/compile` to compile classes
+6. Run `sbt rholangCLI/assembly` to build a stand-alone `.jar` file
 
 ### Command-line usage
 
 ```
-$ ./rho2rbl examples/token.rho
-compiled examples/token.rho to examples/token.rbl
-```
-which is short for:
-```
-$ java -jar target/scala-2.11/rholang-assembly-0.1-SNAPSHOT.jar examples/token.rho 
-compiled examples/token.rho to examples/token.rbl
+$ java -jar rholang-cli/target/scala-2.12/rholangCLI-assembly-0.1.0-SNAPSHOT.jar rholang/examples/token.rho
+<interpreter output follows.>
 ```
 
-### Running rbl with Rosette
-After generating the rbl:
-1.  (cd ../rosette && ./build.sh)
-2.  (cd ../rosette && ./run.sh ../rholang/token.rbl)
+The interpereter can also be run as a REPL. Currently it won't accept multiline input, so each line must be a fully formed term.
 
 ## What's working, what's broken:
 See [the bugtracker](https://rchain.atlassian.net/projects/RHOL/issues/RHOL-95?filter=allopenissues) for an up-to-date list of known issues.
 ### The bad
 In general:
-  * Quoting doesn't work when sending processes.
-  * Similarly running a received process doesn't work.
-  * Listening on channels not created by new doesn't work in general, with a few exceptions
-  * Destructuring as a part of a pattern in a for or a contract doesn't work.
   * Conditional input doesn't work.
-  * We currently don't support multiple-arity receives. We support multi-arity sends, and we will shortly support arity-matching. When that lands, the mismatch will cause no forward progress. Right now it causes a crash in rosette.
   * 0-arity send and receive is currently broken.
-  * Whether a variable holds a name or a value is muddled. In order to do arithmetic, you must use the name directly, not `*name`. See the examples. As we implement destructuring in inputs, this will be straightened out.
+  * Matching is incomplete. We won't currently destructure receives or matches. We will match expressions.
+  * We don't pre-evaluate match cases. So matching 7 + 8 as a pattern currently doesn't work. Instead, you must match against 15.
+  * There is work in progress to support stdout and stderr, but it hasn't landed yet.
+  * There is also work to support native functions. It hasn't landed yet.
 ### The good
 Several working examples have been included in the examples/ directory, and the tests in tests/ also work. If you run into something that doesn't work, check the bugtracker to see if it's a known issue, and if not, feel free to file a bug. We want rholang to be a useful programming environment.
```

Comment:
Based on this line, I guess that means we should clean up the `examples` folder to only have 0.2-compatible examples

Reply:
Or make an 0.2 examples directory, and change the text.

Reply:
Yeah, keeping old examples in an 0.1 folder and putting new examples in an 0.2 folder is a good idea. That's what I did for my linking examples in `rholang/examples/linking`. Not sure how much of that work you want to include in this PR. Maybe just create the folder and also create a Jira ticket to populate it? You can assign the ticket to me if you want.